From 8696305c1efcea42a2227f8efaa7e1e73c71d32c Mon Sep 17 00:00:00 2001
Message-Id: <8696305c1efcea42a2227f8efaa7e1e73c71d32c.1615856156.git.liu.denton@gmail.com>
In-Reply-To: <cover.1615856156.git.liu.denton@gmail.com>
References: <cover.1615856156.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Fri, 12 Mar 2021 02:09:36 -0800
Subject: [PATCH 6/7] ci/run-static-analysis.sh: make check-builtins
To: Git Mailing List <git@vger.kernel.org>
Cc: Johannes Schindelin <Johannes.Schindelin@gmx.de>,
    Junio C Hamano <gitster@pobox.com>

To ensure that any newly introduced builtins are in git.c command list
and that no duplicate script files exist, run the 'check-builtins'
target as part of static-analysis.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 ci/run-static-analysis.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ci/run-static-analysis.sh b/ci/run-static-analysis.sh
index 65bcebda41..1ae122fc70 100755
--- a/ci/run-static-analysis.sh
+++ b/ci/run-static-analysis.sh
@@ -26,7 +26,7 @@ then
 	exit 1
 fi
 
-make hdr-check ||
+make hdr-check check-builtins ||
 exit 1
 
 save_good_tree
-- 
2.31.0.rc2.261.g7f71774620

